auth-web/src/assets/scss/theme.scss

```diff
@@ -17,6 +17,7 @@ $BCgovBlue2: #8099b3;
 $BCgovBlue3: #4d7094;
 $BCgovBlue4: #26527d;
 $BCgovBlue5: #003366; // PRIMARY BLUE
+$BCgovBlue5O: #01336626; // LOW OPACITY PRIMARY BLUE
```

Comment:
This variable is not part of the BC Gov palette so I'd use a different name and move it to a section containing our own colours, as we've done in other UI projects.

Reply:
Will rename and relocate, definitely agree!

I did pull this from either our Create or Edit UI however, might have to fix that there too 😁

Reply:
Updated in next commit